src/component/common/Accordion.jsx

```diff
@@ -0,0 +1,26 @@
+import AccordionLayout from "./AccordionLayout";
+import { useState } from "react";
+const tags = [];
+function Accordion() {
+  const [isOpen, setisOpen] = useState(false);
```

Comment:
I dont see the need to keep the state variables and handleToggle function here since they are just used in the AccordionLayout component, so just use them there itself

Reply:
I'll make the necessary changes.